src/SFA.DAS.ApprenticeFeedback.Application/Commands/CreateApprenticeFeedback/CreateApprenticeFeedbackHandler.cs

```diff
@@ -34,11 +34,44 @@ public async Task<CreateApprenticeFeedbackResponse> Handle(CreateApprenticeFeedb
 
             var validAttributes = await _apprenticeFeedbackRepository.GetAttributes();
 
+            var validAttributesNames = validAttributes.Select(attribute => attribute.AttributeName).ToList();
+
+            var providedAttributesNames = request.FeedbackAttributes.Select(attribute => attribute.Name).ToList();
+
+            string attributesProvided = string.Empty;
+
+            foreach (var attribute in request.FeedbackAttributes)
```

Comment:
For this creation of the list of attributes, have a look at the String.Join method.